cmd/fleetctl/package_test.go

```diff
@@ -25,12 +25,12 @@ func TestPackage(t *testing.T) {
 	// run package tests, each should output their respective package type
 	// fleet-osquery_0.0.3_amd64.deb
 	runAppForTest(t, []string{"package", "--type=deb", "--insecure"})
-	info, err := os.Stat("fleet-osquery_0.0.3_amd64.deb")
+	info, err := os.Stat("fleet-osquery_0.0.5_amd64.deb")
 	require.NoError(t, err)
 	require.Greater(t, info.Size(), int64(0)) // TODO verify contents
 	// fleet-osquery-0.0.3.x86_64.rpm
 	runAppForTest(t, []string{"package", "--type=rpm", "--insecure"})
-	info, err = os.Stat("fleet-osquery-0.0.3.x86_64.rpm")
+	info, err = os.Stat("fleet-osquery-0.0.5.x86_64.rpm")
```

Comment:
Should we either check the latest version from TUF or make this check less specific?